app/views/applications/new.html.erb

```diff
@@ -40,6 +55,30 @@
 
   <div class="form_field">
     <p>
+      <%= f.check_box :terms_and_conditions %>
+      <%= f.label :terms_and_conditions, "Terms and Conditions" %>
+    </p>
+    <p>
+      This diversity program is aimed to help underrepresented groups in tech to attend and lowering the barriers for them. This includes but isn’t limited to: women, people of color, LGBTQIA+ people, and disabled people. Your privacy, details, answers remain protected, we will raffle out the tickets. We will not share any data with those, who sponsor these tickets. This application is for one conference ticket only. The ticket includes
```

Comment:
Could you change to: 
Your privacy, details and answers remain protected; we will raffle out the tickets. We will not share any data with those who sponsor these tickets.